sys/src/9/ufs/ufs_harvey.h

```diff
@@ -79,6 +69,60 @@ typedef struct ComponentName {
 enum vtype { VNON, VREG, VDIR, VBLK, VCHR, VLNK, VSOCK, VFIFO, VBAD, VMARKER };
 typedef enum vtype Vtype;
 
+
+
+
+/*
+ * Vnode flags.
+ *	VI flags are protected by interlock and live in v_iflag
+ *	VV flags are protected by the vnode lock and live in v_vflag
+ *
+ *	VI_DOOMED is doubly protected by the interlock and vnode lock.  Both
+ *	are required for writing but the status may be checked with either.
+ */
+//#define	VI_MOUNT	0x0020	/* Mount in progress */
+//#define	VI_DOOMED	0x0080	/* This vnode is being recycled */
+//#define	VI_FREE		0x0100	/* This vnode is on the freelist */
+//#define	VI_ACTIVE	0x0200	/* This vnode is on the active list */
+//#define	VI_DOINGINACT	0x0800	/* VOP_INACTIVE is in progress */
+//#define	VI_OWEINACT	0x1000	/* Need to call inactive */
+
+#define	VV_ROOT		0x0001	/* root of its filesystem */
+//#define	VV_ISTTY	0x0002	/* vnode represents a tty */
```

Comment:
Hopefully we'll get to the point where we can just delete a lot of this stuff :)